components/multi-platform-controller/staging-downstream/host-config.yaml

```diff
@@ -89,3 +89,9 @@ data:
   host.ppc1.user: "root"
   host.ppc1.secret: "internal-stage-ibm-ssh-key"
   host.ppc1.concurrency: "4"
+
+  host.ibm-gpu-intel64.address: "10.130.81.14"
+  host.ibm-gpu-intel64.platform: "linux-ibm-gpu/intel64"
```

Comment:
one minor thing, shouldn't this be amd64 not intel64?

Reply:
its intel machine not AMD so I thought lets make it intel64.

Reply:
The architecture is called amd64 though

Reply:
yes, let me change it